uui/components/pickers/PickerInput.tsx

```diff
@@ -14,7 +14,12 @@ import { PickerItem } from './PickerItem';
 const pickerHeight = 300;
 const pickerWidth = 360;
 
-export type PickerInputProps = SizeMod & IHasEditMode & {};
+export type PickerInputProps = SizeMod & IHasEditMode & {
+    /**
+    * Defines an id for inner input
+    */
+    id?: string;
```

Comment:
Let's change to -
/** HTML ID attribute for the toggler input */

Reply:
fixed